tests/test_ert_hooks.py

```diff
@@ -0,0 +1,86 @@
+import os
+import sys
+import shutil
+import subprocess
+
+import pytest
+
+try:
+    from ert_shared.plugins.plugin_manager import ErtPluginManager
+except ImportError:
+    pytest.skip(
+        "ERT is not installed, skipping hook implementation tests.",
+        allow_module_level=True,
+    )
```

Comment:
I'm a little bit worried that if the module structure of ERT changes, this test will just be skipped and everything appears green. Maybe do something like this instead?
```python
try:
    import ert_shared
except ImportError:
    pytest.skip(
        "ERT is not installed, skipping hook implementation tests.",
        allow_module_level=True,
    )

from ert_shared.plugins.plugin_manager import ErtPluginManager
```

Reply:
Fixed.